.github/workflows/release.yml

```
@@ -18,34 +18,32 @@ jobs:
         id: get_version
         run: echo ::set-output name=VERSION::$(echo $GITHUB_REF | cut -d / -f 3)
 
+      - name: Login to DockerHub
+        uses: docker/login-action@v1 
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
```

No need to store username in secrets - it is not really secret, otherwise it will replaced with `***` everywhere in logs

```suggestion
          username: hellofreshtech
```